CombineUnloadAIDriver.lua

```diff
@@ -2066,13 +2066,6 @@ function CombineUnloadAIDriver:followChopperThroughTurn()
 		if self.course:getCurrentWaypointIx() > self.combineToUnload.cp.driver.course:getCurrentWaypointIx() then
 			self:hold()
 		end
-		-- follow course, make sure we are keeping distance from the chopper
-		-- TODO: or just rely on the proximity sensor here?
-		local combineSpeed = (self.combineToUnload.lastSpeedReal * 3600)
-		local speed = combineSpeed + MathUtil.clamp(d - (self.minDistanceFromWideTurnChopper - self.settings.combineOffsetZ:get()),
-			-combineSpeed, self:getFieldSpeed())
-		self:setSpeed(speed)
```

Comment:
We still need to set the speed to some value in each loop, as if some other algorithm sets it once to 0 for instance, the vehicle will never start moving again (there's a little explanation in the AIDriver.lua documentation part)
It would be safer to have setl:setSpeed(combineSpeed) in there.

Reply:
Will do, next commit will have these changes in

Reply:
@MddMBorg how is it going ?
Could you adjust the Code ?